index.js

```diff
@@ -168,6 +171,9 @@ export default class Barcode extends PureComponent {
         <Surface height={this.props.height} width={this.state.barCodeWidth}>
           <Shape d={this.state.bars} fill={this.props.lineColor} />
         </Surface>
+        if (typeof(this.props.text) != "undefined") {
+          <Text style={{color: this.props.textColor, width: this.state.barCodeWidth, textAlign: "center"}} >{this.props.text}</Text>
+        }
```

Comment:
The syntax is wrong here.

```diff
-        if (typeof(this.props.text) != "undefined") {
-          <Text style={{color: this.props.textColor, width: this.state.barCodeWidth, textAlign: "center"}} >{this.props.text}</Text>
-        }
+        {typeof(this.props.text) !== 'undefined' && (
+          <Text style={{color: this.props.textColor, width: this.state.barCodeWidth, textAlign: 'center'}} >{this.props.text}</Text>
+        )}
```

Reply:
Yes, it has been fixed last Friday, the fix PR get merged yesterday too. I haven't pushed the update.(having a call with HSM..)